packages/core/getBusyTimes.ts

```diff
@@ -211,3 +211,120 @@ export async function getBusyTimes(params: {
 }
 
 export default getBusyTimes;
+
+// this funcationality cannot be imolpemented in getBusyTime because it has optional and eventTypeId and while searching for bookings it doesnt use eventTypeId filter which can fetch unnecessary bookings
+export async function getEventBookingsForPeriod(params: {
```

Comment:
We can implement this functionality using `getBusyTimes` function by
- adding `checkGivenEventOnly` - a `boolean` param, which if enabled will only get bookings where `eventId === eventTypeId`
- provide `credentials = []` and `selectedCalendars = []`